src/ServiceControl.Transports.SqlServer/SqlServerQuery.cs

```diff
@@ -147,8 +148,12 @@ protected override void InitialiseCore(FrozenDictionary<string, string> settings
     public static class SqlServerSettings
     {
         public static readonly string ConnectionString = "SqlServer/ConnectionString";
-        public static readonly string ConnectionStringDescription = "A single database connection string that will provide at least read access to all queue tables.";
+
+        public static readonly string ConnectionStringDescription =
+            "Database connection string that will provide at least read access to all queue tables.";
```

Comment:
If there's no database specified (ie it's not a "single" db connection) then how would SC know which database/catalog to use?

Reply:
it would error, and not start

Reply:
The connection string requires the initial catalog to be set

Reply:
So why remove the word "single" from the description then?

Reply:
because there is no "multiple"